.changeset/witty-carrots-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+"@saleor/app-sdk": patch
+---
+
+Marked `/middleware`s as deprecated in favor of `/handlers`
```

Comment:
You are merging this to main branch, which is 1.0.0 pre and will have breaking change. You can just remove them now, no need to deprecation.

Deprecation makes sense in v0.x branch

Reply:
my bad, I sent wrong base branch 👍🏻